concepts/settings/project-roles.mdx

```diff
@@ -12,15 +12,22 @@ Once a project is created, you can invite others to collaborate:
 
 1. Go to the project settings.
 2. Invite members by entering their email addresses.
-3. Assign roles to each member:
-   - **Admin**: Full access to the project.
-   - **User**: Read-only access to the project.
+3. Assign roles to each member.
 
 <Frame caption="Invite a collaborator">
   <img src="/images/members.jpg" alt="Invite a collaborator" />
 </Frame>
 
-Contact us to create more fine-grained roles.
+### Role Based Access Control (RBAC)
+
+Literal AI supports Role-Based Access Control (RBAC) to streamline and secure access to project resources within teams. RBAC ensures that team members have appropriate permissions aligned with their roles, enhancing both security and efficiency in managing project activities.
+
+Project administrators hold the responsibility of assigning and managing roles for all team members. Each role is designed with a distinct set of permissions, ensuring that users can perform their duties effectively without overstepping into areas beyond their scope.
+
+**Supported roles:**
+- **Admin**: Admins have full, unrestricted access to the entire project. They can manage all aspects of the project, including adding and removing team members, modifying project settings, and accessing and altering all project resources. Admins are responsible for the overall governance of the project, ensuring smooth operations and compliance with organizational policies.
+- **AI Engineer**: AI Engineers have specific access tailored to their technical responsibilities. They can manage observability data, generate and manage API keys, and create and modify prompts and rules necessary for the AI's evaluation. However, they do not have permissions to alter project settings or manage team member roles. AI Engineers focus on the technical implementation and maintenance of AI models observability, ensuring optimal performance.
+- **Domain Expert**: Domain Experts have a specialized set of permissions, allowing them to read observability data. They can also score and tag data, providing insights based on their expertise. Domain Experts have limited access to broader project resources and do not manage API keys, or project settings. Their role is to interpret data and provide feedback that enhances the project's data, ensuring outputs align with domain-specific requirements.
```

Comment:
Again, i find this too verbose and GPT-y. I can't scan the page and get the information i need cause it's lost in all the prose. It seems like Mintlify supports [tables](https://mintlify.com/docs/list-table), i'm sure a simple one with :heavy_check_mark:s and :red_circle:s would be advantageous here.

Reply:
Do you think of that? https://langfuse.com/docs/rbac#scopes-per-role

Reply:
not a fan, i'd rather have something simple like (non exhaustive)

|                        | Admin | AI Engineer | Domain expert |
|------------------------|-------|-------------|---------------|
| Alter project settings | ✅     | 🔴           | 🔴             |
| Invite users           | ✅     | 🔴           | 🔴             |
| Create project key     | ✅     | ✅           | 🔴             |
| View observability     | ✅     | ✅           |  🔴           |
| Score / Tag data       | ✅     | ✅           | ✅             |

Reply:
What is the status on that?

Reply:
@willydouhard I would rather avoid introducing a map of roles this way as it's either:
- not extensive
- not maintained

I first suggest to keep it a bit verbose and refine once we get to more than 3 roles, and we need to provide more granularity.

Reply:
A table is clearer. I get the valuable information directly

Reply:
yeah sounds fine !